composer.json

```diff
@@ -45,21 +45,22 @@
         "ext-pdo_pgsql": "*"
     },
     "require-dev": {
-        "php": ">=8.0",
+        "php": ">=7.4",
         "ext-phar": "*",
         "ext-tokenizer": "*",
         "ext-xmlwriter": "*",
         "phpstan/phpstan": "^1.10",
         "phpstan/phpstan-strict-rules": "^1.5",
-        "squizlabs/php_codesniffer": "^3.7"
+        "squizlabs/php_codesniffer": "^3.9"
     },
     "scripts": {
         "php-lint": "find . -type d -name 'vendor' -prune -o -name '*.php' -print0 | xargs -0 -n1 -P4 php -l 1>/dev/null",
         "phtml-lint": "find . -type d -name 'vendor' -prune -o -name '*.phtml' -print0 | xargs -0 -n1 -P4 php -l 1>/dev/null",
         "phpcs": "phpcs . -s",
         "phpcbf": "phpcbf . -p -s",
         "phpstan": "phpstan analyse --memory-limit 512M .",
-        "phpstan-next": "phpstan analyse --level 9 --memory-limit 512M $(find . -type d -name 'vendor' -prune -o -name '*.php' -o -name '*.phtml' | grep -Fxvf ./tests/phpstan-next.txt | sort | paste -s -)",
+        "phpstan-next": "phpstan analyse --memory-limit 512M -c phpstan-next.neon .",
+        "phpstan-third-party": "phpstan analyse --memory-limit 512M -c phpstan-third-party.neon .",
```

Comment:
For a future PR: it would be nice to add that during the GitHub Actions for `generate.php` to e.g. report warnings or even exclude failing extensions

Reply:
This memory limit thing is for GitHub Actions btw?

Reply:
It used to fail even on my machine without it, but I have not tested recently, and now that our code base has much fewer errors, it might work with the default value

Reply:
Ahh I figured it was the other way around. xD